tox.ini

```diff
@@ -0,0 +1,40 @@
+[tox]
+skipsdist = true
+
+# These envs are run in order if you just run `tox`
+envlist =
+    format # runs auto-formatter - black and isort
+    mypy   # type checking
+    py     # tests
+
+[base]
+deps =
+    -r requirements.txt
+
+[testenv:py]
+deps =
+    pytest
+    pytest-cov
+    responses
+    {[base]deps}
+
+commands =
+    pytest
+
+[testenv:format]
```

Comment:
I don't think we should be using `tox` as a framework for running other tools besides doing actual testing.